plugins/code-syntax-highlight/webpack.config.js

```diff
@@ -44,28 +44,34 @@ function getOutputConfig(isProduction, isCDN, isAll, minify) {
 }
 
 function getExternalsConfig(isProduction, isCDN, isAll) {
-  if (isProduction && !isAll) {
-    if (isCDN) {
-      return [
-        {
-          'highlight.js/lib/highlight': {
-            commonjs: 'highlight.js',
-            commonjs2: 'highlight.js',
-            amd: 'highlight.js',
-            root: ['hljs']
-          }
+  const cdnBundle = isProduction && isCDN && !isAll;
+  const npmBundle = isProduction && !isCDN;
+  const demoBundle = !isProduction && !isAll;
+
+  // Only the code-syntax-highlight provide a bundle file
+  // with an external dependency on cdn
```

Comment:
이 문장만으로는 의미 전달이 잘 안되는 것 같네요. 이유도 명확히 적어두면 좋을 것 같습니다.
The code-syntax-highlight plugin should provide a CDN bundle without the highlight.js dependency 
so that users can inject their own highlight.js instance.

Reply:
뒤에 `when using only selected languages` 추가하는건 어떨까요??
사실상 CDN에서는 전체 언어를 포함한 `all` 버전도 제공하는데 이 경우에는 주입을 안시키거든요.
`선택한 언어만 사용하려는 경우`에 주입을 시키는 버전을 사용하니까 설명 추가하는게 좋겠네요ㅎ